tests/integration/targets/ec2_instance_metadata_options/tasks/main.yml

```diff
@@ -44,6 +46,7 @@
         metadata_options:
           http_endpoint: enabled
           http_tokens: optional
+          http_put_response_hop_limit: 4
         wait: false
       register: metadata_options_update
       ignore_errors: true
```

Comment:
We probably shouldn't ignore errors here since this task will need to complete successfully for the next task to pass.